tests/e2e/tests/Map/DefaultLegendLayersInTriggeredFlood.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from '@playwright/test';
+import DashboardPage from 'Pages/DashboardPage';
+import MapComponent from 'Pages/MapComponent';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { TriggerDataSet } from 'testData/testData.enum';
+
+import { FloodsScenario } from '../../../../services/API-service/src/scripts/enum/mock-scenario.enum';
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    FloodsScenario.Trigger,
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+// https://app.qase.io/project/IBF?previewMode=side&suite=3&tab=&case=37
+test(
+  qase(
+    37,
+    '[Trigger] Map layer: "Flood extent" and "Exposed population" should be active by default',
```

Comment:
What is the exact purpose of this test. I often get mixed up by filenames vs test descriptions here vs full test cases in Qase. Is in this case the test just about the layers being present in the legend? And with legend do you mean the lower-left thing or the lower-right thing? 

Either way, I think it makes sense to test all together. So test (1) are the layers in the layer overview (lower-right), (2) are they in the legend (lower-left), and (3) are they in the map? I'm not clear on if this is happening now either in this test or in multiple tests.